src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -11,9 +11,9 @@ import './PursuancePage.css';
 class PursuancePage extends Component {
 
   componentWillMount() {
-    const { setCurrentPursuance, match } = this.props;
-    const currentPursuanceId = Number(match.params.pursuanceId);
-    setCurrentPursuance(currentPursuanceId);
+    let { setCurrentPursuance, match, currentPursuanceId } = this.props;
+      currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
```

Comment:
`match.params.pursuanceId` should always be defined, right? Is the `||` needed, then?

Reply:
I think there was an edge case where it would be `NaN` after creating a pursuance and then going back to a different pursuance, but I'm having a hard time duplicating that instance so I'll just remove it

Reply:
I see.  I'd think that `match.params.pursuanceId` would always be defined, and by a value like `"1"` that can be turned into a number, since that value comes from the URL, but if `PursuancePage` is ever being loaded some other way -- which I don't think it is? -- then I can see how that could happen.

Reply:
If you run `npm run lint`, is this whitespace still there?

Reply:
yup